### ISSUE TITLE
snakemake documentation: 404 error for referenced file

### ISSUE THREAD

**Issue by nashda** [snakemake/snakemake#1315]
**Snakemake version**
N/A

**Describe the bug**
[https://snakemake.readthedocs.io/en/stable/executor_tutorial/azure_aks.html#running-the-workflow](https://snakemake.readthedocs.io/en/stable/executor_tutorial/azure_aks.html#running-the-workflow)
On the snakemake doc there is a referenced zip of support files, namely the yaml configurations for the AKS workflow. The link to the zip file returns a 404 and the contents of those files are not in the documentation to be recreated. Please provide one or the other.
![Capture](https://user-images.githubusercontent.com/22985797/147400756-1035cd7d-8404-4dc7-8ee1-27e8229976d1.PNG)

**Logs**

**Minimal example**

**Additional context**

**Comment by aryarm** [comment]
hmm looks like that documentation was copied from [this blog post](https://andreas-wilm.github.io/2020-06-08-snakemake-on-ask/#getting-the-snakefile), so the file is here:

https://andreas-wilm.github.io/data/2020-06-08/snakedir.zip

maybe @andreas-wilm might be able to comment on this issue?

**Comment by andreas-wilm** [comment]
That's right. The issue sounds familiar. If I remember correctly this was pointed out and fixed before!?

**Comment by aryarm** [comment]
oh, yeah! It's a duplicate of #1074

**Comment by schnea** [comment]
docs page no longer exists
